frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -447,59 +442,12 @@ const Profile: NextPage = () => {
     </section>
   );
 
-  const bottomPremiumSection =
-    profile.has_premium ||
-    !isPeriodicalPremiumAvailableInCountry(runtimeData.data) ? null : (
-      <section className={styles["bottom-banner"]}>
-        <div className={styles["bottom-banner-wrapper"]}>
-          <div className={styles["bottom-banner-content"]}>
-            {isPhonesAvailableInCountry(runtimeData.data) ? (
-              <>
-                <Localized
-                  id="footer-banner-premium-promo-headine"
```

Comment:
If you haven't done so already, probably good idea to mark these strings as deprecated (/remove them, if this is merged) in the l10n repo, to avoid burdening the volunteer localisers more.